app/tests/spec/views/form.js

```diff
@@ -161,6 +161,12 @@ define(function (require, exports, module) {
         assert.isFalse(view.isFormEnabled());
         assert.isTrue(view.$('button').hasClass('disabled'));
       });
+
+      it('checks if enablig timer is set', function () {
```

Comment:
we also need a test that the form gets activated after the timeout. 

Here's an example of changing the timeout for testing purposes: https://github.com/mozilla/fxa-content-server/blob/master/app/tests/spec/views/confirm.js#L251 

You can use that to make the timeout really short and validate that your new code works.